### PR TITLE
Project completed

### PR DESCRIPTION
**Pull request by dennis-every**
This pull request is for review and approval to merge the dev branch into the main branch. The following tasks were completed:

- Project setup with create-react-app and linters
- Add routes and navigation with react-router
- Add Axios to fetch data from the API
- Add Redux store to retrieve and store the data
- Add tests with react-testing-library and Jest
- Add styles with bootstrap and react icons
- Deploy demo app on Render
- Add readme with a live link to the demo

### CODE REVIEW

File: src/components/StocksContainer.jsx
Comment (brytebee) [inline src/components/StocksContainer.jsx:1]: This component is required to contain a way the user can search the data rendered. Kindly add this.

image 

You could place this search box strategically in your design.

image

image

File: src/components/StocksContainer.scss
Comment (brytebee) [inline src/components/StocksContainer.scss:1]: You are required to keep to the design template given for the project.

There are inconsistencies in the render of your boxes and their dimensions. Kindly fix.

The `font-size` of the titles and figures varies, kindly take note and implement. 

**NOTE**
If the titles length affect the sizes of check design, you could `slice` them to be uniform throughout the page, then the whole title would be visible in the details page.


Your Design | Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @dennis-every👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Proper flow used for your version control.✔️
- App functions as it should.✔️
- Great video presentation. Bravo 👏👏👏
    - Good video length.✔️
    - Demos app and mentions the part you're proud of.✔️
- App is hosted and link is added to README.✔️


##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes ♻️ 



_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions from the previous review are done! 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/StocksContainer.scss
Comment (dennis-every) [reply src/components/StocksContainer.scss:1]: Ok, tks for the feedback!

**Comment by dennis-every** [comment]
Ok, tks for the feedback!

**Review by omar-muhamad — COMMENTED** [review]
# Approved ✔
### Hi @dennis-every,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**